bot/wuestbot.py

```diff
@@ -1,6 +1,374 @@
-#!/usr/bin/env python3
+import os
+import logging
+from traceback import print_exc
 
-from app import run
+import flask
+import requests
+from telebot import TeleBot, types
+from dotenv import load_dotenv
 
-if __name__== "__main__":
-	run()
\ No newline at end of file
+from app.database import db_objects
+
+# set logger
+logging.basicConfig(filename='log.txt', encoding='utf-8', level=logging.DEBUG)
+
+# load env variables from .env file
+load_dotenv()
+API_TOKEN = os.getenv('API_TOKEN', '')
+HOST = os.getenv('HOST', '')
+PORT = os.getenv('PORT', '8443')
+ADMINS = os.getenv('ADMINS', '')
+
+# URL to let telegram know where we are 
+WEBHOOK_URL = f"https://{HOST}:{PORT}/bot"
+
+bot = TeleBot(API_TOKEN)
+app = flask.Flask(__name__)
+
+img_dir = os.path.abspath('../bot/app/img')
+
+# Process webhook calls
+@app.route(f'/bot', methods=['POST'])
+def webhook():
+    if flask.request.headers.get('content-type') == 'application/json':
+        json_string = flask.request.get_data().decode('utf-8')
+        update = telebot.types.Update.de_json(json_string)
+        bot.process_new_updates([update])
```

Comment:
What does `process_new_updates` do exacty? 🤔

Reply:
Takes the json objects from the webhook, sorts it into message categories and hands those over to specific handler functions. basically a big bunch of [if statements](https://github.com/eternnoir/pyTelegramBotAPI/blob/87bce0bce149bc6b3aa67f1252a3115273d94c3f/telebot/__init__.py#L325)

Reply:
Ah it's a library function